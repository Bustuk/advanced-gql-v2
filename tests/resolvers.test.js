const resolvers = require('../src/resolvers');

describe('resolvers', () => {
  test('feed', async () => {
    const result = resolvers.Query.feed(null, null, {
      models: 
      { Post: 
        { 
          findMany: jest.fn(() => [{id: 1, message: 'hello', createdAt: 12345839, likes: 20, views: 300}]) 
        } 
      }
    })

    expect(result).toMatchSnapshot()
  })
})