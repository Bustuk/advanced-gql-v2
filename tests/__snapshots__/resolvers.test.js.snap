// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolvers feed 1`] = `
Array [
  Object {
    "createdAt": 12345839,
    "id": 1,
    "likes": 20,
    "message": "hello",
    "views": 300,
  },
]
`;
